ci_framework/roles/ci_setup/tasks/packages.yml

```diff
@@ -15,46 +15,50 @@
     name: "{{ cifmw_ci_setup_packages }}"
     state: latest
 
+- name: Gather version of openshift client
+
+  register: _oc_version
+  environment: >-
+    {{
+      {'PATH': cifmw_path}
+      if cifmw_path is defined else
+      {}
+    }}
+  ansible.builtin.command:
+    cmd: "oc version -o yaml"
```

Comment:
in fact, we want the client, really, meaning we want:
`oc version --client -o yaml`
Else, it will try to query any known/configured remote server for its APIs versions, and will fail in case of unresponsive server. It will also wait until a timeout hits, making the whole thing unstable and, usually, throwing a wrong answer in such case (i.e. client installed, but no server answers -> crash).